rex/external/standard_attrs.json

```diff
@@ -0,0 +1,35 @@
+{
+    "time_index": {
+        "standard_name": "time",
+        "long_name": "time",
+        "calendar": "proleptic_gregorian"
+    },
+    "latitude": {
+        "standard_name": "latitude",
+        "units": "degree_north"
+    },
+    "longitude": {
+        "standard_name": "longitude",
+        "units": "degree_east"
+    },
+    "temperature": {
+        "standard_name": "air_temperature",
+        "units": "C"
+    },
+    "windspeed": {
+        "standard_name": "wind_speed",
+        "units": "m s-1"
+    },
+    "winddirection": {
+        "standard_name": "wind_to_direction",
```

Comment:
I used that but I don't know if that is true. Better check it with Brandon.

Reply:
Yea, honestly it's probably supposed to be just `"wind_direction"`. I basically have not checked the data in the JSON files yet. Part of finalizing this PR will include updating the data in these files

Reply:
wind dir is the *from* direction in WTK/sup3r

Reply:
@grantbuster Thanks for following up on this. Can I use `"wind_from_direction"` as the `standard_name` here then? 

![image](https://github.com/user-attachments/assets/a105626d-217e-4e68-bb8f-2daee3a9f89e)

Reply:
Yes i think that is most appropriate!